packages/react-components/react-nav-preview/stories/Nav/NavDefault.stories.tsx

```diff
@@ -1,4 +1,21 @@
 import * as React from 'react';
-import { Nav, NavProps } from '@fluentui/react-nav-preview';
+import { Nav, NavProps, NavGroup } from '@fluentui/react-nav-preview';
+import { makeStyles } from '@fluentui/react-components';
 
-export const Default = (props: Partial<NavProps>) => <Nav {...props} />;
+const useStyles = makeStyles({
+  root: {},
+});
```

Comment:
I know this probably isn't finished yet... but unless you're planning to add styling to this root div in a future PR, you should remove this makeStyles call (and possibly remove the div itself).